### PR TITLE
CHE-1754: improve ide loader

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?

Loader Improving  (it changed loader widget content and its styles).
#1754

anim

Signed-off-by: Oleksii Orel oorel@codenvy.com

### CODE REVIEW

File: dashboard/src/app/ide/ide.html [hunk 1]
```diff
@@ -15,12 +15,13 @@
...
     <div ng-hide="ideCtrl.selectedWorkspace">No workspace selected. Unable to open IDE</div>
-    <div layout="column" class="ide-page-loader-content" ng-show="ideCtrl.selectedWorkspace">
-      <div class="spinner">
-        <div class="rect1"></div>
-        <div class="rect2"></div>
-        <div class="rect3"></div>
-      </div>
+    <div class="ide-page-loader-content" ng-show="ideCtrl.selectedWorkspace">
+        <div>
+          <span>Loader idea</span>
```
Comment (benoitf) [inline dashboard/src/app/ide/ide.html]: Could you attach an animated gif ?
Because I'm not sure we should display "loader idea" text

**Review by benoitf — CHANGES_REQUESTED** [review]

File: dashboard/src/app/ide/ide.html (hunk 1 above)
Comment (slemeur) [reply dashboard/src/app/ide/ide.html]: We should definitely not show it.

**Review by benoitf — APPROVED** [review]
ok for technical part
for UI I let @slemeur

File: .gitignore [hunk 2]
```diff
@@ -24,6 +24,10 @@ test-output/
 *.o
 *.so
 
+# TypeScript built source #
+###########################
+dashboard/src/**/*.js
```
Comment (benoitf) [inline .gitignore]: I don't have on my side the typescript built source in the same folder so I'm not sure about this exclude ?

**Review by ashumilova — APPROVED** [review]

**Review by akurinnoy — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (olexii4) [inline .gitignore]: Sorry. It needed for some built in compiler. It very useful to go through to resources. I have removed this commit.

**Comment by olexii4** [comment]
@benoitf Sorry. It needed for some built in compiler. It very useful to go through to resources. I have removed this commit.

**Comment by benoitf** [comment]
@olexii4 is that builtin compiler could generate in a separate folder ? because I think it should be difficult to have all files in same folders no ?
Also is that sourceMaps isn't working ?

**Comment by olexii4** [comment]
@benoitf  It is configurable. I have removed this commit.

**Comment by slemeur** [comment]
It looks okay for me.
We need to try to use this loader in other areas - where we has the small vertical bars loader.
In this case, I would think that the Che logo must have to be white.

**Comment by slemeur** [comment]
So looking at the changes in the animation, I'm -1 for changing the vertical bar from other places than the splash-screen. (ie, the left navbar and in the IDE)

**Comment by olexii4** [comment]
anim

**Review by slemeur — APPROVED** [review]

**Pull request merged**